web/app/view/DevicesController.js

```diff
@@ -117,5 +122,9 @@ Ext.define('Traccar.view.DevicesController', {
         if (position !== undefined) {
             this.getView().getSelectionModel().deselectAll();
         }
+    },
+
+    onUpdateDevice: function () {
+        this.view.fireEvent('selectionchange',this.view.getSelectionModel());
```

Comment:
Can't we just call `onSelectionChange` method directly? Also, space is missing between arguments.